shared/components/EventList.js

```diff
@@ -37,7 +37,12 @@ class EventList extends Component {
         <MapButton
```

Comment:
we can change the name of this component, because we also use this in others component not only for map!